metadata.rb

```diff
@@ -17,4 +17,4 @@
 supports 'scientific'
 supports 'oracle'
 
-depends 'ark', '~> 5.0'
+depends 'ark', '~> 6.0'
```

Comment:
Can we please change it to the following so it's a bit more flexible for users?
```suggestion
depends 'ark', '>= 6.0'
```